configuration.md

```diff
@@ -46,20 +46,32 @@ Default: 512
 ### eventer
 
 ```
-<eventer>
-  <config>
-    <ssl_dhparam1024_file/>
-    <ssl_dhparam2048_file/>
-  </config>
-</eventer>
+<include file="irondb-eventer.conf" />
+```
+
+Libmtev eventer system configuration is included from a separate file controlled
+by the vendor. Changes to this file will be overwritten by package updates.
+
+Details about the included configuration can be found in
+the [libmtev eventer documentation](http://circonus-labs.github.io/libmtev/config/eventer.html).
+Specifically, the `ssl_dhparam*_file` configurations are set to null to disable
+automatic Diffie-Hellman parameter generation at startup. IRONdb does not
+utilize TLS by default, though this capability is present in libmtev.
+
+### irondb-modules
```

Comment:
I might add something to explain what "modules" are since we have not yet exposed any of the Lua stuff via IRONdb.  Users won't know what a "module" is or why they would want one.  For now we could just say that these modules will be used in the future to provide advanced features, and that more information will be available when those features are ready.  Something like that...